backend/apis/parse-docket-pdf.api.js

```diff
@@ -6,8 +6,10 @@ const { generateFlagJson } = require("./generate-flag-json.utils");
 
 app.post("/api/docket-pdfs", (req, res) => {
   let busboy;
+  const busBoyConfig = { headers: req.headers };
```

Comment:
Here we can add additional configuration such as limiting number of files. Do we know the max number of dockets a user can upload?

Reply:
I think it's hard to set a limit on number of dockets. for now it is more reasonable to set limit on `fileSize`. It has been set to 10mb which should be plenty.